ext/src/Bigint.c

```diff
@@ -37,17 +46,7 @@ to_double(zval *result, php_driver_numeric *bigint TSRMLS_DC)
 static int
 to_long(zval *result, php_driver_numeric *bigint TSRMLS_DC)
 {
-  if (bigint->data.bigint.value < (cass_int64_t) LONG_MIN) {
```

Comment:
True. Since this is used for the `toInt()` method should we indicate that the `toDouble()` or `toString()` method should be used to get the value contained within the `Bigint`?